extras/docker/Dockerfile

```diff
@@ -19,92 +25,38 @@ ARG HEADLESS=false
 #
 ######################################################################################## 
 
-FROM node:${NODE_VERSION} as builder
+FROM ${BUILDER} as builder
 ARG GITHUB_ACCOUNT
 ARG GITHUB_REPOSITORY
 ARG DOWNLOAD
 ARG SOURCE_BRANCH
+ARG PACKAGE_MANAGER
 
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+# hadolint ignore=DL3002
+USER root
 
 ENV PYTHONUNBUFFERED=1
```

Comment:
This `ENV` looks very suspicious - I guess it is an artifact from an older python-based version. Should the line be deleted?

> Setting `PYTHONUNBUFFERED` to a non empty value ensures that the python output is sent straight to terminal (e.g. your container log) without being first buffered and that you can see the output of your application (e.g. django logs) in real time.